### DISCUSSION TITLE
Help! Unable To Decrypt Using New GPG Fingerprint

### DISCUSSION THREAD

**Discussion by dcplaya** [getsops/sops#940] [Q&A]
Hoping I can get some help. I am unable to find any info about this error in my searches of the internet.

I have a current set of encrypted files and a corresponding `.sops.yaml` file with 2 GPG fingerprints. One is for some K8s software called Flux, the other is my own Yubikey fingerprint. Both Flux and my dev environment (Windows laptop using WSL2) are able to decrypt and encrypt my secrets. 

My issue comes when I try and add a new fingerprint. Below are the steps I have done to add a new fingerprint.

1.  Created a new GPG key
2. Copied new GPG key's fingerprint into `.sops.yaml`
3. Ran `sops updatekey` on the secrets
4. Verified the new fingerprint is in the secrets encrypted details
5. Copied secret to a different PC. A Ubuntu 21.04 server edition inside a VM
6. Exported the private and public key of the new fingerprint from the Windows dev laptop
7. Imported the keys into the Ubuntu VM
8. Tried to decrypt the secrets on the Ubuntu VM and get the following error
```sh
root@ubuntu:~/projects$  sops -d cluster/base/cluster-secrets.yaml
WARN: no KMS client could be accessed:
gpg: encrypted with 4096-bit RSA key, ID XXXXXXXX, created 2021-04-15
      "k8s (Flux) email@email.com"
gpg: decryption failed: No secret key
gpg: encrypted with RSA key, ID YYYYYYYYY
gpg: decryption failed: No secret key
gpg: encrypted with 4096-bit RSA key, ID ZZZZZZZZZ, created 2021-09-16
      "Ubuntu (VM) email@email.com"
PANIC: Checksum verification failed!
expected AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
but got  BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB
```
I have replaced possible identifying info with placeholders.

Can anyone give me some help on what I am doing wrong?

**Comment by zradeg** [comment]
I have the same issue. Have you found any solution?
